README.md

```diff
@@ -1,125 +1,106 @@
-# lookout [![Build Status](https://travis-ci.org/src-d/lookout.svg)](https://travis-ci.org/src-d/lookout) [![GoDoc](https://godoc.org/gopkg.in/src-d/lookout?status.svg)](https://godoc.org/github.com/src-d/lookout)
+lookout
+[![Build Status](https://travis-ci.org/src-d/lookout.svg)](https://travis-ci.org/src-d/lookout)
+[![GoDoc](https://godoc.org/gopkg.in/src-d/lookout?status.svg)](https://godoc.org/github.com/src-d/lookout)
+[![Code Coverage](https://codecov.io/github/src-d/lookout/coverage.svg)](https://codecov.io/github/src-d/lookout)
+[![Go Report Card](https://goreportcard.com/badge/github.com/src-d/lookout)](https://goreportcard.com/report/github.com/src-d/lookout)
+![development](https://svg-badge.appspot.com/badge/stability/development?color=D6604A)
+=======
 
 A service for assisted code review, that allows running custom code Analyzers on pull requests.
 
-# SDK
-
-If you are developing an Analyzer, please check [SDK documentation](./sdk/README.md).
-
-It includes a curl-style binary `lookout-sdk` that allows to trigger Analyzers directly, without launching a full lookout server.
 
 # Installation
 
 ```bash
 $ go get github.com/src-d/lookout/...
 ```
 
-# Dependencies
+It will globally install `lookoutd`, `lookout-sdk`, and `dummy`.
+
+
+## Dependencies
 
-The included [`./docker-compose.yml`](./docker-compose.yml) allows to start all dependencies using [Docker Compose](https://docs.docker.com/compose/) 
+**lookout** will run custom Analyzers to perform the analysis of the PR.
+It also needs a running instance of:
 
-* [bblfshd](https://github.com/bblfsh/bblfshd), on `localhost:9432`
-* [PostgreSQL](https://www.postgresql.org/), on `localhost:5432` password `postgres`
+* [bblfshd](https://github.com/bblfsh/bblfshd) to parse files into [UAST](https://doc.bblf.sh/uast/uast-specification.html)
+* [PostgreSQL](https://www.postgresql.org) to store the analysis events and logs
 
-Clone the repository, or download [`./docker-compose.yml`](./docker-compose.yml)
 
 # Usage
 
-To trigger the analysis on an actual pull request of a GitHub repository you will need [GitHub access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/).
+The **lookout** configuration is defined by the `config.yml` file; you can use the template [`config.yml.tpl`](config.yml.tpl) to create your own config file. You will find more information about it in the [docs about how to configure lookout](docs/configuration.md)
+
+
+To trigger the analysis on any pull request of a GitHub repository you will need a GitHub authentication as it is described in the [docs about how to authenticate with GitHub](docs/configuration.md#basic-auth)
+
```

Comment:
Probably a nit-pick, but 2 sentences above does not end with period.

Reply:
wow!! that was neat!
thanks for it.